api/src/integrations/integration-utils.js

```diff
@@ -92,12 +92,12 @@ exports.getRecords = async function(url, options = undefined) {
       throw new Error('coreLogin - unable to log in to Core API.');
     }
 
-    return payload.access_token;
-  }
+    return payload;
+  };
```

Comment:
changed this to return the whole object because it includes the ttl for the token